index.js

```diff
@@ -16,6 +18,25 @@ module.exports = {
   included() {
     this._super.included.apply(this, arguments);
 
+    let peerDependencies = require('./package').peerDependencies;
+    let checker = VersionChecker.forProject(this.project);
+
+    if (!checker.check(peerDependencies)) {
+      let packageNames = Object.keys(peerDependencies).map(
+        (name) => peerDependencies[name]
+      );
+      let hasYarnLock = this.project.has('yarn.lock');
+
+      let installMessage = hasYarnLock
+        ? `yarn add --dev ${packageNames.join(' ')}`
+        : `npm install --dev ${packageNames.join(' ')}`;
+
+      throw new SilentError(
+        `ember-qunit now requires that \`qunit\` and \`@ember/test-helpers\` are \`devDependencies\` of the project. Please run:\n\t${installMessage}`
+      );
+    }
+
+    // TODO: remove these `this.import` statements when the app is using using Embroider or ember-auto-import
```

Comment:
@ef4 - I'm not 100% sure how to do this, curious if you have thoughts...

(note: I still plan to land this change since it doesn't _require_ automatic Embroider/ember-auto-import interop initially)